java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -1653,6 +1653,42 @@ void testGroupByCountWithNulls() {
     }
   }
 
+  @Test
+  void testGroupByCountWithNullsIncluded() {
+    try (Table t1 = new Table.TestBuilder().column(null, null,    1,    1,    1,    1)
+            .column(   1,    1,    1,    1,    1,    1)
+            .column(   1,    1, null, null,    1,    1)
+            .column(   1,    1,    1, null,    1,    1)
+            .build()) {
+      try (Table tmp = t1.groupBy(0).aggregate(count(1,true), count(2,true), count(3,true));
+           Table t3 = tmp.orderBy(Table.asc(0, true));
+           HostColumnVector groupCol = t3.getColumn(0).copyToHost();
+           HostColumnVector countCol = t3.getColumn(1).copyToHost();
+           HostColumnVector nullCountCol = t3.getColumn(2).copyToHost();
+           HostColumnVector nullCountCol2 = t3.getColumn(3).copyToHost()) {
+        // verify t3
+        assertEquals(2, t3.getRowCount());
+
+        // compare the grouping columns
+        assertTrue(groupCol.isNull(0));
+        assertEquals(groupCol.getInt(1), 1);
+
+        // compare the agg columns
+        // count(1)
+        assertEquals(countCol.getInt(0), 2);
+        assertEquals(countCol.getInt(1), 4);
```

Comment:
I think this one also includes nulls? Would be good to add a count somewhere (not of the first col obviously) that doesn't count nulls in this test, for completeness.